### ISSUE TITLE
IndexError: list index out of range on Nexus 5X Android 6.0.1

### ISSUE THREAD

**Issue by curdriceaurora** [Turnsole/cookie-butter#1]
```
...
Round 59 done, now have 2197 frames.
Traceback (most recent call last):
  File "generate_frametime_graphs.py", line 164, in <module>
    main(sys.argv[1:]) 
  File "generate_frametime_graphs.py", line 158, in main
    fps = scrape_display_info(args.device)
  File "generate_frametime_graphs.py", line 44, in scrape_display_info
    return subprocess.check_output([cmd], shell=True).split()[0]
IndexError: list index out of range
```

**Comment by Turnsole** [comment]
If you don't mind, it'd help me to know: 
- what device you were looking at 
- what command you used

**Comment by curdriceaurora** [comment]
Nexus 5X 6.0.1
python generate_frametime_graphs.py com.package.name 7 example_graph UDID

Found another device, tablet with 5.1 and it worked fine.

**Comment by Turnsole** [comment]
Lucky me, I've got one of those handy, and can reproduce. Thanks for the report.

**Comment by Turnsole** [comment]
Just pushed an update which should take care of that.

**Comment by curdriceaurora** [comment]
That fixed it for the 6.0.1 device and doesn't break it for the 5.1 device. Thanks. Closing the issue.

**Comment by peteroid** [comment]
I was facing the same issue. My phone is Xiaomi Note 4 and it's on Android 6. In my adb command, there is no "PhysicalDisplayInfo". So I just pull all the fps out and select the top one for a quick fix. 

```python
cmd = "adb {} shell dumpsys display | grep -oE 'fps=[0-9\.]+' | head -n 1 | cut -c 5-".format(target)
```
